tests/explicit_import.rs

```diff
@@ -0,0 +1,30 @@
+#[macro_use]
+extern crate num_derive;
+extern crate num_traits as panini;
```

Comment:
Were you hungry when you wrote this? :slightly_smiling_face:

Reply:
Yeah that's probably why I was lazy and didn't make a test crate haha.